elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -550,6 +556,12 @@ export class PeerPool extends EventEmitter {
 		return peers.filter(peer => peer.state === ConnectionState.OPEN);
 	}
 
+	public getDisconnectedPeers(): ReadonlyArray<Peer> {
+		const peers = [...this._peerMap.values()];
```

Comment:
The `_peerMap` should never contain disconnected peers because we automatically remove peers as soon as they disconnect. We need a different strategy to compute the disconnected list. We may need to use the PeerInfos from the address book.

Also we need to decide what is meant by disconnected peer; maybe it's a peer we were connected to before (triedPeers which we are not currently connected to)? Or should we include newPeers (which we haven't yet connected to before)?

Reply:
Addressed.